integration/appaccess/appaccess_test.go

```diff
@@ -559,7 +560,12 @@ func TestTCP(t *testing.T) {
 
 // TestTCPLock tests locking TCP applications.
 func TestTCPLock(t *testing.T) {
-	pack := Setup(t)
+	clock := clockwork.NewFakeClockAt(time.Now())
```

Comment:
If using `Setup` without specifying a clock is known to be flaky, can we eliminate that as a possibility by making it require a clock? Otherwise I fear we'll just reintroduce the flakiness in future tests.